### ISSUE TITLE
[stackedArea] Graphs should fail gracefully when presented with badly formatted data

### ISSUE THREAD

**Issue by aicioara** [palantir/plottable#1848]
Easiest to comment on the JS fiddle
http://jsfiddle.net/d2te4yct/6/

- Graphs 1, 2 and 3 should be the same. "0" should be 0 and "s" should default to 0
- Graph 4 is not rendered until #1838 is merged. For discussion sake, in this jsfiddle http://jsfiddle.net/d2te4yct/7/ you can see that the blue area is not rendered at all (on the right hand side). Ideally we should have that area filled with `blue`

**Comment by aicioara** [comment]
For convenience
![screen shot 2015-04-16 at 12 24 26 pm](https://cloud.githubusercontent.com/assets/3248682/7189393/8dd5e838-e433-11e4-88ae-3759854a2f49.png)

**Comment by aicioara** [comment]
Since graph 4 may not be expressive enough 
![screen shot 2015-04-16 at 10 59 59 am](https://cloud.githubusercontent.com/assets/3248682/7187772/d6868198-e427-11e4-9c80-837ed4fc43d8.png)
Ideally we want this 
![screen shot 2015-04-16 at 11 01 08 am](https://cloud.githubusercontent.com/assets/3248682/7187781/e7ab5264-e427-11e4-9c16-5b01dae801e0.png)

**Comment by aicioara** [comment]
@jtlan Did you say that you are creating a couple other issues related to this one? Can you reference them here/

**Comment by aicioara** [comment]
After discussions about use cases, we concluded the following 
- Plots should fail silently when presented with badly formatted data and, in general, badly formatted points should not be displayed (hence graph 4 and 3 are acceptable)
- "0" should be coerced to 0
- `undefined` and `null` should behave the same when fed as data to a plot. As per bullet point 1, they should not be displayed

From 1-3 above, the conclusion is that graph 2 is wrong, but as a side talk we concluded that null and undefined should behave the same (opening new ticket for this)

**Comment by aicioara** [comment]
Ticket finished, but blocked by https://github.com/palantir/plottable/pull/1844
